### PR TITLE
Add new topic to describe how to allow multiple admin login

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request adds a new topic to describe how to allow multiple admin login

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [x] Other feature set, please specify:

whatsnew
Updated [Admin Sign In](https://docs.magento.com/user-guide/stores/admin-signin.html) to add information about allowing login from multiple devices.

### CODE REVIEW

**Comment by dobooth** [comment]
Any reason we shouldn't just add this to https://docs.magento.com/user-guide/stores/admin-signin.html?

**Comment by ZhovnirMaria** [comment]
Hi, @dobooth 
Hmm, you're right, if you don’t mind, I can just add this as a new block to the existing Admin Login page. When I started working on this topic, it seemed to me that this could be taken out as a separate topic.

**Comment by ZhovnirMaria** [comment]
Hi, @dobooth. I have added this topic as a part of the https://docs.magento.com/user-guide/stores/admin-signin.html page. Please check, thanks 😊

File: src/stores/admin-signin.md [hunk 1]
```diff
@@ -93,4 +93,21 @@ _Sign Out_
 
 The Sign In page displays a message that you are logged out. We recommend you sign out of the _Admin_ any time you leave your computer unattended.
 
+## Allow Multiple Admin Login
```
Comment (leslietilling) [inline src/stores/admin-signin.md:96]:
```suggestion
## Allow multiple Admin logins
```

File: src/stores/admin-signin.md [hunk 2]
```diff
@@ -93,4 +93,21 @@ _Sign Out_
...
+
+The admin manages the orders, customers, products, shipping, and payments functionality. The default configuration is set to not allowing the admin account sharing for security purposes. Multiple admin users can manage the store faster and distribute the workload for better efficiency.
```
Comment (leslietilling) [inline src/stores/admin-signin.md:98]: This function actually determines if an account can be logged in at the same time from different devices. 

```suggestion
The Admin provides access to manage the orders, customers, products, shipping, and payments functionality. The default configuration is set to disallow multiple logins for an Admin user account as a security best practice. However, you can change this setting to allow Admin users to be logged in from multiple devices if that is needed to accommodate your business workflows.
```

File: src/stores/admin-signin.md [hunk 3]
```diff
@@ -93,4 +93,21 @@ _Sign Out_
...
+1. On the _Admin_ sidebar, go to **Stores** > **Configuration**.
+
+    ![]({% link images/images/multiple-admin-login.png %}){: .zoom}
+    _Multiple Admin Login_
+
+1. Under **Advanced**, select **Admin**.
+
+1. Expand the **Security** section.
+
+1. Set `Yes` in the **Admin Account Sharing** field.
```
Comment (leslietilling) [inline src/stores/admin-signin.md:109]:
```suggestion
1. For **Admin Account Sharing**, select `Yes`.
```

File: src/stores/admin-signin.md [hunk 4]
```diff
@@ -93,4 +93,21 @@ _Sign Out_
 
 The Sign In page displays a message that you are logged out. We recommend you sign out of the _Admin_ any time you leave your computer unattended.
 
+## Allow Multiple Admin Login
+
+The admin manages the orders, customers, products, shipping, and payments functionality. The default configuration is set to not allowing the admin account sharing for security purposes. Multiple admin users can manage the store faster and distribute the workload for better efficiency.
+
+1. On the _Admin_ sidebar, go to **Stores** > **Configuration**.
```
Comment (leslietilling) [inline src/stores/admin-signin.md:100]:
```suggestion
1. On the _Admin_ sidebar, go to **Stores** > _Settings_ > **Configuration**.
```

File: src/stores/admin-signin.md [hunk 5]
```diff
@@ -93,4 +93,21 @@ _Sign Out_
...
+
+    ![]({% link images/images/multiple-admin-login.png %}){: .zoom}
+    _Multiple Admin Login_
+
+1. Under **Advanced**, select **Admin**.
```
Comment (leslietilling) [inline src/stores/admin-signin.md:105]:
```suggestion
1. In the left panel, expand **Advanced** and choose **Admin**.
```

File: src/stores/admin-signin.md [hunk 6]
```diff
@@ -93,4 +93,21 @@ _Sign Out_
...
+
+1. Expand the **Security** section.
```
Comment (leslietilling) [inline src/stores/admin-signin.md:107]:
```suggestion
1. Expand ![Expansion selector]({% link images/images/btn-expand.png %}) the **Security** section.
```

File: src/stores/admin-signin.md [hunk 7]
```diff
@@ -93,4 +93,21 @@ _Sign Out_
...
+
+1. Set `Yes` in the **Admin Account Sharing** field.
+
+1. Click **Save** the configuration.
```
Comment (leslietilling) [inline src/stores/admin-signin.md:111]:
```suggestion
1. Click <span class="btn">Save Config</span>.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
edits needed

File: src/stores/admin-signin.md (hunk 7 above)
Comment (ZhovnirMaria) [reply src/stores/admin-signin.md]: Done, thank you🙏😊

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**